CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - N/A
 
+## [2.5.0] - 2020-02-21
+
+### Added
+- Introduced `Repository.sync` API for syncing repositories
+
+### Fixed
+- Fixed a bug that export an empty maintenance report would crash.
```

Comment:
This stuff is already under 2.4.0 and seems to have been duplicated under 2.5.0 for some reason.